### ISSUE TITLE
VMTK fails to build since recent ITK changes.

### ISSUE THREAD

**Issue by chir-set** [vmtk/SlicerExtension-VMTK#55]
I'm building SlicerVMTK on Arch Linux, with Slicer at e4b8a88, using clang 13.0.1 and cmake 3.22.2.

VMTK itself won't build without the patch below. It's probably not optimal, but demonstrates the problem. I have not yet tested for runtime issues.

Please inform if it should be addressed in master. If not, I can cope with my local branch.

Thank you.

```
diff --git a/vtkVmtk/Segmentation/itkAnisotropicDiffusionVesselEnhancementImageFilter.txx b/vtkVmtk/Segmentation/itkAnisotropicDiffusionVesselEnhancementImageFilter.txx
index 29bc0c6..f5adaa6 100644
--- a/vtkVmtk/Segmentation/itkAnisotropicDiffusionVesselEnhancementImageFilter.txx
+++ b/vtkVmtk/Segmentation/itkAnisotropicDiffusionVesselEnhancementImageFilter.txx
@@ -406,7 +406,13 @@ AnisotropicDiffusionVesselEnhancementImageFilter<TInputImage, TOutputImage, TVes
   this->GetMultiThreader()->SingleMethodExecute();

   // Resolve the single value time step to return
-  dt = this->ResolveTimeStep(str.TimeStepList, str.ValidTimeStepList);
+  std::vector<itk::uint8_t> xValidTimeStepList;
+  std::vector<bool>::iterator it;
+  for (it = str.ValidTimeStepList.begin(); it != str.ValidTimeStepList.end(); it++)
+  {
+    xValidTimeStepList.push_back((itk::uint8_t) *it);
+  }
+  dt = this->ResolveTimeStep(str.TimeStepList, xValidTimeStepList);
   //delete [] str.TimeStepList;
   //delete [] str.ValidTimeStepList;

```

**Comment by lassoan** [comment]
I was able to fix SlicerVMTK build in Slicer Preview Release. Please test and close this issue if everything works well.

**Comment by chir-set** [comment]
Fixed in #56.
